scripts/202306xx_populate_paymenttype_default/add_fixed_payment_type_default.js

```diff
@@ -4,31 +4,87 @@
 Add PaymentType.Fixed to all forms with existing payments_field object
 */
 
-// Count number of forms that contains payments_field property
-db.getCollection('forms')
-  .find({ payments_field: { $exists: true } })
-  .count()
+// Batch 1 2022
+const queriesModifiedIn2022 = {
+  payments_field: { $exists: true },
+  lastModified: { $gte: new ISODate('2022-01-01T00:00:00.000+0000') },
+  lastModified: { $lt: new ISODate('2023-01-01T00:00:00.000+0000') },
+}
 
-// Count number of forms that contains payments_field.payment_type property.
-// Expected to be 0
-db.getCollection('forms')
-  .find({ "payments_field.payment_type": { $exists: true } })
-  .count()
+const queriesModifiedIn2023Q1 = {
+  lastModified: { $gte: new ISODate('2023-01-01T00:00:00.000+0000') },
+  lastModified: { $lt: new ISODate('2023-04-01T00:00:00.000+0000') },
+}
 
-/*
-Create payments_field.payment_type from existing payments_fields
+const queriesModifiedIn2023AprMay = {
+  lastModified: { $gte: new ISODate('2023-04-01T00:00:00.000+0000') },
+  lastModified: { $lt: new ISODate('2023-06-01T00:00:00.000+0000') },
+}
+
+const queriesModifiedIn2023JunOnwards = {
+  lastModified: { $gte: new ISODate('2023-06-01T00:00:00.000+0000') },
+}
+
+function executePaymentTypeInsertionOnQuery(dateQuery, dryRun = true) {
+  console.log(dateQuery)
+  // Count number of forms that contains payments_field property
+  const formsWithPaymentsField = db
+    .getCollection('forms')
+    .find({
+      payments_field: { $exists: true },
+      ...dateQuery,
```

Comment:
`lastModified` isn't indexed right? Would this affect the query time 

pinging @mayying